### PR TITLE
Support regions

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

- Update cloud-info-provider for region support
- Add region configuration for CNAF

---

**Related issue :**

### CODE REVIEW

File: sites/INFN-CLOUD-CNAF.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: https://cloud-api-pub.cr.cnaf.infn.it:5000/v3/
 gocdb: INFN-CLOUD-CNAF
-protocol: openid
+region: sdds
```
Comment (gwarf) [inline sites/INFN-CLOUD-CNAF.yaml:3]: So I lack all the background but this changes looks a bit weird to me, but maybe it's just normal.

Comment (enolfc) [reply sites/INFN-CLOUD-CNAF.yaml:3]: The protocol `openid` is the default, so was redundant information.
The actual change is adding the region in there

Comment (enolfc) [reply sites/INFN-CLOUD-CNAF.yaml:3]: BTW, this information is actually also available in GOC, see https://goc.egi.eu/portal/index.php?Page_Type=Site&id=2745, but reading it from there was more complicated than this change at the moment

**Comment by enolfc** [comment]
@CatalinCondurache this is something that may be nice if you can take a look too

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**